src/main/java/stsjorbsmod/cards/cull/Goosebumps.java

```diff
@@ -0,0 +1,45 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Goosebumps extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Goosebumps.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE = 3;
+    private static final int MULTI = 3;
+    private static final int UPGRADE_MULTI = 1;
```

Comment:
Probably unnecessary repetition on my part after the other reviews, but I would note _PLUS in the name for this upgrade. Perhaps rename to ATTACK_COUNT and UPGRADE_PLUS_ATTACK_COUNT?